src/extension-support/argument-type.js

```diff
@@ -1,8 +1,31 @@
+/**
+ * Block argument types
+ * @enum {string}
+ */
 const ArgumentType = {
+    /**
+     * Numeric value with angle picker
+     */
     ANGLE: 'angle',
+
+    /**
+     * Boolean value with hexagonal placeholder
+     */
     BOOLEAN: 'Boolean',
```

Comment:
Heh - I was wondering if anyone would balk at that. I wrote all the words the way they'd appear in the middle of a sentence, where Boolean should get a capital because it's based on the name of George Boole. My thinking is that either these value strings won't ever matter as long as they're unique, or they could be used in a message like `Invalid value for ${arg.type} argument`, where "Boolean" would be the [technically-correct](https://imgur.com/gallery/PpV0evc) text to insert.